torch_ipex/csrc/cpu/DevOPs.cpp

```diff
@@ -549,6 +549,40 @@ at::Tensor AtenIpexCPUDev::dil_linear(
   return dbl::comm::gen_aten_tensor_by(y);
 }
 
+at::Tensor AtenIpexCPUDev::dil_linear_fuse_relu(
+    const at::Tensor& self,
+    const at::Tensor& weight,
+    const c10::optional<at::Tensor>& bias) {
+  DEBUG("AtenIpexCPUDev::dil_linear\n");
+  CHECK_DNNL_OP_PRE_COND(self);
+  CHECK_DNNL_OP_PRE_COND(weight);
+  TORCH_CHECK(self.dim() >= 2,
+      "dil_linear: input needs to has dim at least 2, input dim ", self.dim());
+
+  // reshape first if input dim is greater than 2 and the reshape will cost a memory copy.
+  auto self_reshaped = self.dim() > 2 ? self.reshape({-1, self.size(self.dim() - 1)}) : self;
+  const dil::tensor x = dbl::comm::try_gen_dil_tensor(self_reshaped);
+  const dil::tensor w = dbl::comm::try_gen_dil_tensor(weight);
+
+  dil::tensor y;
+  if (bias.has_value()) {
+    at::Tensor bias_vec = bias.value();
+    const dil::tensor b = dbl::comm::try_gen_dil_tensor(bias_vec);
+    dil::inner_product_forward::compute(x, w, b, y, /*fuse_relu=*/true);
```

Comment:
please reuse attr parameter, and remove this new parameter

Reply:
Done